### DISCUSSION TITLE
Hi Tomas,  If I can get opportunity to contribute on this repo in any manner please do let me know.

### DISCUSSION THREAD

**Discussion by MdSaleh** [tomasfabian/ksqlDB.RestApi.Client-DotNet#53] [General]
Hi Tomas,  If I can get opportunity to contribute on this repo in any manner please do let me know.

**Comment by tomasfabian** [comment]
Hello @MdSaleh, 
thank you for your kind offer, but currently, there are no open issues or feature requests. 

Regards Tomas

**Comment by tomasfabian** [comment]
Hello @MdSaleh,
Could you please let me know if you're using a Windows or Linux operating system? I'm asking because some of the unit tests are failing under Linux. Your contribution in resolving this issue would be greatly appreciated in case that it is interesting for you. Thank you in advance!
